apps/server/src/utilities/shutdown.service.ts

```diff
@@ -1,15 +1,22 @@
-import { Injectable, Logger, OnApplicationShutdown } from '@nestjs/common';
-import { SysTrayService } from './systray.service';
+import { Injectable, Logger, OnModuleDestroy } from '@nestjs/common';
+import { Subject } from 'rxjs';
 
 @Injectable()
-export class ShutDownService implements OnApplicationShutdown {
-  private readonly logger = new Logger(ShutDownService.name);
+export class ShutDownService implements OnModuleDestroy {
+    private readonly logger = new Logger(ShutDownService.name);
 
-  constructor(private systrayService: SysTrayService) {}
+    private shutdownListener$: Subject<void> = new Subject();
 
-  onApplicationShutdown(signal: string) {
-      this.logger.log(`Application shutting down with signal: ${signal}`);
-      // Handle all graceful kill events
-      this.systrayService.kill();
-  }
+    onModuleDestroy() {
+        this.logger.log('ShutDown Service being destroyed');
```

Comment:
```suggestion
        this.logger.log('Destroying ShutDown Service');
```

Just to match the SysTray service.

Reply:
resolved [88f4d09](https://github.com/flybywiresim/simbridge/pull/58/commits/88f4d09fec86b16a99cdf417d2931e91fb89e74d)